src/controllers/users.js

```diff
@@ -0,0 +1,6 @@
+/* Get all users from Firebase */
+export const getUsers = (req, res) => {
+  // TODO: Use Firebase functions to get all user objects
+  // This will allow for selecting individual users and updating their user roles
```

Comment:
I'm not sure if it's possible, but ideally, we could request all the user objects in Firebase and send that information back to the admin.

Reply:
I think this is possible with `firebase-admin` https://firebase.google.com/docs/auth/admin/manage-users#list_all_users